### PR TITLE
In Detail component

### PR DESCRIPTION
**Pull request by Keqi**
For some reason my styles are not loaded from `src/components/in_detail` directory. Temporarily I put them in `intro_narrative` dir but I would love someone to take a look why this is happening.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,7 +31,7 @@
     "version:minor": "npm run release -- --release-as minor",
     "version:major": "npm run release -- --release-as major"
   },
-  "version": "0.28.4",
+  "version": "0.29.0",
```
Comment (thomasthesecond) [inline package.json]: We're not releasing new versions of Rizzo next like this anymore.

We are using [standard-version](https://github.com/conventional-changelog/standard-version) to help with releases and automatic generation of the CHANGELOG.

That means you'll need to format your commit messages a certain way. Look at the [CONTRIBUTING.md](https://github.com/lonelyplanet/rizzo-next/blob/master/CONTRIBUTING.md) doc for more information.

Comment (thomasthesecond) [reply package.json]: A new version is then released by @jcreamer898 @nickfranciosi or myself.

**Comment by Keqi** [comment]
Hey @thomasthesecond,

Thanks for help. I've squashed my commits and set correct commit description. Can you take a look again?

**Comment by thomasthesecond** [comment]
https://github.com/lonelyplanet/dotcom-pois/issues/1358

**Pull request merged**